src/builtin-addons/core/intl-utils.ts

```diff
@@ -0,0 +1,204 @@
+import * as flat from 'flat';
+import { tokenize, parseTokens, AST } from 'json-parse-ast';
+import * as path from 'path';
+import * as YAML from 'yaml';
+import { Location, Range } from 'vscode-languageserver-types';
```

Comment:
could we use `import { Definition, Location } from 'vscode-languageserver'` instead of `vscode-languageserver-types`? (to streamline with already existing usages)